contribs/gmf/apps/mobile/index.html

```diff
@@ -66,6 +62,9 @@
       <div id="themes" class="slide" data-header-title="Themes">
         <span>Themes</span>
       </div>
+      <div id="login" class="slide" data-header-title="Login">
```

Comment:
Do you really need a container div ?

`<gmf-authentication id="login" class="slide" data-header-title="Login"></gmf-authentication>` is not enough ?